### ISSUE TITLE
Template overrides not working with Ratings & Reviews fields

### ISSUE THREAD

**Issue by zackkatz** [gravityview/GravityView#1076]
See [HS#14878](https://secure.helpscout.net/conversation/587692016/14878), which includes the modified template files.

The code essentially is this:

```
foreach( $this->getEntries() as $entry ) {

$this->setCurrentEntry( $entry );

$this->renderZone( 'columns', array(
'markup' => '<td id="{{ field_id }}" class="{{class}}">{{value}}</td>',
'hide_empty' => false, // Always show <td>
));
}
```

And the Ratings & Reviews fields didn't display. I assume it's an issue with the legacy state.

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/995529792029955/995555131354735)

**Comment by zackkatz** [comment]
I bet this is fixed with 7d9cb217e2fec331ad5b17b0b733644eac2d24eb

**Comment by zackkatz** [comment]
`#RafaClose` 🧟‍♂️
